### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by typebrook**
- Improve English wording
- Surround command, files , terms with `` ` ` ``(backticks)
- In each codeblock, make comment starts from capitalized char
- command/parameter -> option
- Add description about `--translate-tags` from PR #107

### CODE REVIEW

**Comment by yihong0618** [comment]
@Can you use simplified Chinese or add another README for Traditional Chinese～

**Comment by typebrook** [comment]
Sorry, my bad. I don't distinguish simplified/traditional Chinese in my head when typing.

I'll change them to simplified Chinese in `README-CN.md`

**Comment by typebrook** [comment]
DONE
https://github.com/yihong0618/bilingual_book_maker/pull/119/commits/c6bf631

File: README-CN.md [hunk 1]
```diff
@@ -7,50 +7,62 @@ bilingual_book_maker 是一个 AI 翻译工具，使用 ChatGPT 帮助用户制
...
 
 
 ## 使用
 
-1. pip install -r requirements.txt
-2. OpenAI API key，如果有多个可以用英文逗号分隔(xxx,xxx,xxx)，可以减少接口调用次数限制带来的错误
-3. 本地放了一个 animal_farm.epub 给大家测试
+1. `pip install -r requirements.txt`
+2. 使用 `--openai_key` 指定 OpenAI API key，如果有多个可以用英文逗号分隔(xxx,xxx,xxx)，可以减少接口调用次数限制带来的错误。  
+   或者，指定环境变数 `OPENAI_API_KEY` 来略过这个选项。
```
Comment (yihong0618) [inline README-CN.md:20]: 变数 -> 变量

File: README-CN.md [hunk 2]
```diff
@@ -63,10 +75,13 @@ python make_book.py --book_name 'animal_farm.epub' --openai_key sk-XXXXX --api_b
 
 - 任何 issue PR 都欢迎
 - Issue 中有些 TODO 没做的都可以选
-- 提交代码前请先 `black make_book.py`
+- 提交代码前请先執行 `black make_book.py` [^black]
```
Comment (yihong0618) [inline README-CN.md:78]: 執行 -> 执行

**Comment by yihong0618** [comment]
Its always interesting to find the difference about simplified Chinese and Traditional Chinese
image

**Comment by typebrook** [comment]
So true! Read this blog if you are interested:
https://byvoid.com/zht/categories/語言文字/
https://byvoid.com/zht/blog/region-phrases-comparison-transistor/

File: README-CN.md (hunk 2 above)
Comment (typebrook) [reply README-CN.md]: DONE in 1bcb72e

File: README-CN.md (hunk 1 above)
Comment (typebrook) [reply README-CN.md]: Done in 1bcb72e

**Pull request merged**